Dockerfile

```diff
@@ -73,16 +74,18 @@ RUN curl -sL https://github.com/postmodern/chruby/archive/v0.3.9.tar.gz | tar xz
     rm -rf /chruby-* && rm -rf /usr/local/src/* && yum clean all
 
 ## GIT clone manageiq-appliance and self-service UI repo (SSUI)
-RUN git clone --depth 1 https://github.com/ManageIQ/manageiq-appliance.git ${APPLIANCE_ROOT} && \
-git clone --depth 1 https://github.com/ManageIQ/manageiq-ui-self_service.git ${SSUI_ROOT} && \
-ln -vs ${APP_ROOT} /opt/manageiq/manageiq
+RUN mkdir -p ${APP_ROOT} && \
+    mkdir -p ${APPLIANCE_ROOT} && \
+    mkdir -p ${SSUI_ROOT} && \
+    ln -vs ${APP_ROOT} /opt/manageiq/manageiq && \
+    curl -L https://github.com/ManageIQ/manageiq-appliance/tarball/${REF} | tar vxz -C ${APPLIANCE_ROOT} --strip 1 && \
+    curl -L https://github.com/ManageIQ/manageiq-ui-self_service/tarball/${REF} | tar vxz -C ${SSUI_ROOT} --strip 1
 
-## Create approot, ADD miq
-RUN mkdir -p ${APP_ROOT}
-ADD . ${APP_ROOT}
+## Add ManageIQ source from local directory (dockerfile development) or from Github (official build)
+# ADD . ${APP_ROOT}
```

Comment:
@bdunne my original idea with this one was to default to "ADD ." but give the builder the flexibility to use a preferred REF/branch at build-time, satisfying both use cases. If we comment "ADD" then there is no need to use tarballs, we can just call git in Dockerfile. The "ADD ." is pretty popular for building local branches/unmerged code that's why I thought maybe would be a good default but offer alternative options to builders.